introduction/theTechnology.md

```diff
@@ -0,0 +1,59 @@
+<a href="https://www.surfly.com/">![logo](../images/logosmall.png)</a>
+
+# The Technology
+
+So let's dive into the technology behind Surfly. This chapter is dedicated to explaining how Surfly works compared to other solutions, as well as creating a better understanding of our co-browsing software. This will make it easier to implement our API and understand the code as you will become more familiar with the greater concepts behind it.
+
+<a name="coBrowsingVersus"></a>
+##### Co-browsing vs. Screen-sharing{#coBrowsingVersus}
+
+Nowadays there are countless co-browsing and screen-sharing solutions on offer. To select one might be a difficult task as it is sometimes not immediately clear what the differences are between them. First of all, let's filter down to the distinction between co-browsing and screen-sharing.
+
+This is an easy one: unlike co-browsing, screen-sharing does not limit itself to the web browser. With a screen-sharing solution you are able to share your complete desktop. There are some benefits to this, but from a security perspective it's not the safest solution. There's a bond of trust that needs to exist between users, in order for you to feel comfortable to share the contents of your computer with others.
+
+Also, screen-sharing solutions are pixel-based, which means that the controller constantly takes snapshots of the screen and then sends these to the other side in compressed form. The drawback of this approach is that screen updates are slow and of low quality.
+
+What most screen-sharing and co-browsing solutions, other than Surfly, have in common, is that both rely on external software that needs to be installed by both users. This makes it unsuitable for most web situations as people are often unwilling to install extra software that circumvents the browser's security measures.
+
+<a name="jsCoBrowsing"></a>
+##### Javascript Solutions{#jsCoBrowsing}
+
+With Javascript based solutions, a widget is created in which the Javascript of the original page is being loaded. This is a time consuming process since requests from the user in control continuously need to be sent to the website, then to the co-browsing solution, then back to the controlling user as well as the followers.
```

Comment:
This is not necessarily true. JS-based solutions might also send over the changes directly to the co-browsing solution, so that it doesn't affect leader's experience